converters/fromZigbee.js

```diff
@@ -953,8 +961,50 @@ const converters = {
             };
         },
     },
+    ecozy_hvacThermostat_attReport: {
```

Comment:
Could you (or maybe @tb-killa or @Chrischi-) have a quick look at this? I think I resolved this, but now my zigbee2mqtt crashes, whenever I receive a message from the thermostat (see https://github.com/Koenkk/zigbee2mqtt/issues/518#issuecomment-442457138)

Reply:
try this:
```

    thermostat_devChange: {
        cid: 'hvacThermostat',
        type: 'devChange',
        convert: (model, msg, publish, options) => {
            return {
                localTemp: precisionRound(msg.data.data['localTemp'], 2) / 100,
                occupiedHeatingSetpoint: precisionRound(msg.data.data['occupiedHeatingSetpoint'], 2) / 100,
                setpointChangeSource: msg.data.data['setpointChangeSource'],
                setpointChangeAmount: msg.data.data['setpointChangeAmount'] / 100,
                setpointChangeSourceTimeStamp: msg.data.data['setpointChangeSourceTimeStamp'],
            };
        },
},

```

`msg.data.data['localTemp']` instead of `msg.data.data.localTemp`

Reply:
Works! Thanks a lot 👍